common/src/main/scala/org/apache/celeborn/common/util/PbSerDeUtils.scala

```diff
@@ -114,12 +114,41 @@ object PbSerDeUtils {
         fileMeta.setIsWriterClosed(true)
         fileMeta
     }
-    new DiskFileInfo(
-      userIdentifier,
-      pbFileInfo.getPartitionSplitEnabled,
-      meta,
-      pbFileInfo.getFilePath,
-      pbFileInfo.getBytesFlushed)
+    if (pbFileInfo.getStorageType != 0) {
```

Comment:
Could you simplify this implementation for better readability like following?
```
var storageType: StorageInfo.Type = null
if (pbFileInfo.getStorageType == 0) {
  if (Utils.isHdfsPath(pbFileInfo.getFilePath)) {
        storageType = StorageInfo.Type.HDFS
  } else if (Utils.isOssPath(pbFileInfo.getFilePath)) {
        storageType = StorageInfo.Type.OSS
  } else if (Utils.isS3Path(pbFileInfo.getFilePath)) {
        storageType = StorageInfo.Type.S3
  }
} else {
   storageType = StorageInfo.typesMap.get(pbFileInfo.getStorageType)
}
new DiskFileInfo(userIdentifier,
          pbFileInfo.getPartitionSplitEnabled,
          meta,
          pbFileInfo.getFilePath,
          storageType,
          pbFileInfo.getBytesFlushed)
```

Reply:
thanks, I have updated the code.